src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -193,4 +193,23 @@ public void waitForOtherOptionTextToBeClickable(int qnNumber, int responseNumber
         waitForElementToBeClickable(element);
     }
 
+    public void verifyVisibilityAndCloseMoreInfoAboutEqualShareModal() {
+        WebElement moreInfoAboutEqualShareModalLink = browser.driver.findElement(By.id("more-info-equal-share-modal-link"));
+        click(moreInfoAboutEqualShareModalLink);
+        WebElement moreInfoAboutEqualShareModal = browser.driver.findElement(By.id("more-info-equal-share-modal"));
+        waitForElementVisibility(moreInfoAboutEqualShareModal);
+        closeModal();
+    }
+
+    public void closeModal() {
+        WebElement closeButton = browser.driver.findElement(By.className("close"));
+        waitForElementToBeClickable(closeButton);
+        click(closeButton);
+        try {
+            Thread.sleep(1000);
```

Comment:
@HirdayGupta yes sure. Also, I did not intend for this PR to wait for #3317 to be fixed. My idea was to have this be merged first as is, and then fix the `sleep` call later in a future PR, however replacing this now in this PR would work too. 👍

As for #3317 in general, I believe it does not apply solely to modals, but to other places where `sleep` is used, and a solution for that might be quite different from the one proposed here, which is targeted at modals only. As such, perhaps it might be better not to "railroad" the person working on the PR into a predetermined solution.